src/store/server/getters.ts

```diff
@@ -159,4 +159,25 @@ export const getters: GetterTree<ServerState, any> = {
 
         return interfaces
     },
+
+    getThrottledStateFlags: (state) => {
+        let flags = state.throttled_state.flags.filter((flag: string) => flag !== '?')
+        /*let flags = [
+            'Under-Voltage Detected',
+            'Frequency Capped',
+            'Currently Throttled',
+            'Temperature Limit Active',
+            'Previously Under-Volted',
+            'Previously Frequency Capped',
+            'Previously Throttled',
+            'Previously Temperature Limited',
+        ]*/
```

Comment:
Is this commented out code still needed?

Reply:
It's nice to have this comment for testing.